app/components/whatsapp-button.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+import { AiOutlineWhatsApp } from 'react-icons/ai';
+
+const WhatsappButton = () => {
+  const whatsappNumber = '351939299776'; // Replace with the actual number
```

Comment:
The WhatsApp number can be an `env` variable to, if necessary, change it in the future doesn't need a new deployment.

Reply:
I dont believe it will be necessary since it is public in other platforms